bot/resources/tags/venv.md

```diff
@@ -1,21 +1,20 @@
 **Virtual Environments**
 
-Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible.
+Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible, allowing dependency conflicts to be resolved, and your program requirements to be found easily.
```

Comment:
I think the wording here could be a little better. Perhaps `preventing cross-project dependency conflicts and easy isolation of requirements`?

Reply:
That's a lot cleaner - done :+1: